mkosi/config.py

```diff
@@ -4489,14 +4489,14 @@ def summary(config: Config) -> str:
                     Shim Bootloader: {config.shim_bootloader}
               Unified Kernel Images: {config.unified_kernel_images}
         Unified Kernel Image Format: {config.unified_kernel_image_format}
-        Unified Kernel Image Addons: {line_join_list(config.pe_addons)}
                             Initrds: {line_join_list(config.initrds)}
                     Initrd Packages: {line_join_list(config.initrd_packages)}
```

Comment:
Shouldn't these 2 options go down to the section with "Kernel Modules Initrd"? Seems a bit off to have them separate from all the other initrd options.